microsite/blog/2022-11-15-linux-foundation-introduction-to-backstage-course.md

```diff
@@ -0,0 +1,16 @@
+---
+title: The Linux Foundation launches its Introduction to Backstage course
+description: The new Linux Foundation course is available for free on edX and is designed to help individuals understand how to map Backstage to their organization needs
+author: Jorge Lainfiesta, Roadie
+authorURL: https://www.linkedin.com/in/jrlainfiesta/
+---
+
+Backstage continues to grow in [popularity and maturity](https://roadie.io/blog/backstage-consolidating-its-role-in-the-cloud-native-ecosystem/), with industry leaders not only adopting the framework but actively participating in the community through contributions and commercial offerings. One of the most recent contributions comes from the Linux Foundation, [launching an introductory course](https://training.linuxfoundation.org/blog/23107/) aimed at DevOps engineers or professionals working on Developer Productivity or Developer Experience. The course is available for free on edX, with a paid option for the certified track.
+
+[![Introduction to Backstage: Developer Portals Made Easy (LFS142x): Enroll!](assets/22-11-15/Introduction-to-Backstage-Developer-Portals-Made-Easy-2-768x432.png)](https://www.edx.org/course/introduction-to-backstage-developer-portals-made-easy)
+
+<!--truncate-->
+
+The course, [Introduction to Backstage: Developer Portals Made Easy (LFS142x)](https://www.edx.org/course/introduction-to-backstage-developer-portals-made-easy), starts discussing the benefits of adopting a Developer Portal. Then, it dives into what is Backstage, its main features (Catalog, Scaffolder, TechDocs), and how to map its capabilities to your organization. At last, it provides tips on how to connect with the community to ease up your adoption journey.
+
+The course author, [Jorge Lainfiesta](https://www.linkedin.com/in/jrlainfiesta/), is a Technical Marketing Manager at [Roadie](https://roadie.io). Jorge has a background in software engineering (PayPal) and digital communication (UCLA). He’s been working around Backstage since it was open sourced by Spotify and co-hosts community initiatives like the Backstage Users Unconference.
```

Comment:
We also need to place Lainfiesta into our vale accept.txt file to please or spell checker :) https://github.com/backstage/backstage/blob/master/.github/vale/Vocab/Backstage/accept.txt

Reply:
Ohh gotcha! I added it now! Thanks!